changelog/fragments/remove-new-cmd-for-helm.yaml

```diff
@@ -0,0 +1,17 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      The `operator-sdk new` command no longer supports scaffolding new Helm projects with the `--type=Helm` flag.
+      To scaffold new projects, users are expected to use `operator-sdk init --plugins=helm.operator-sdk.io/v1` as part of the
+      [new CLI]((https://sdk.operatorframework.io/docs/new-cli)) for Helm operators.
+
+      See `operator-sdk init --plugins=helm.operator-sdk.io/v1 -h` and the following doc on how to scaffold a new project
+      https://sdk.operatorframework.io/docs/helm/quickstart/. To migrate existing projects to the new layout see:
+      https://sdk.operatorframework.io/docs/helm/migration.
```

Comment:
This is much too long for a one-liner changelog line. But the other day we talked about keeping these removal fragments short and sweet to remind ourselves what to write when we do actually write the 1.0 release notes.

Reply:
Remove and keep TBD as suggested.